Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift

```diff
@@ -30,6 +30,9 @@ public struct DeviceInfo: Equatable, Encodable {
     public let physicalMemory: String?
     public let cpuCount: String?
 
+    /// Hostname of the device
+    public let kernelHostname: String?
```

Comment:
Given that `hardwareInfoHarvester.kernelHostname` returns non-optional value:
```suggestion
    public let kernelHostname: String
```